csm_web/scheduler/admin.py

```diff
@@ -583,7 +583,7 @@ class OverrideAdmin(admin.ModelAdmin):
     fields = ("date", "spacetime", "overriden_spacetime")
     ordering = ("-date",)
 
-    list_filter = ("spacetime___day_of_week", "spacetime__section__course")
+    list_filter = ("spacetime__day_of_week", "spacetime__section__course")
```

Comment:
funny how we forgot to change the underscore in the second filter

Reply:
We didn't forget, the field `section` (no leading underscore) was always public. If you try to refer to a non-existent field in the admin code (or any other code that gets loaded upon running the dev server) Django will complain and not serve you any pages until it's fixed.